content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -1239,6 +1239,49 @@ The following `ExecCredential` manifest describes a cluster information sample.
 {{% /tab %}}
 {{< /tabs >}}
 
+## Get self subject attributes
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+With the `Self Subject Attributes Review` API, you can be aware of user attributes represented by your current environment (kubeconfig file or in-cluster config).
+
+For convenience, the `kubectl auth who-am-i` command is present. Executing this command will produce the following output (yet different user attributes will be shown):
+
+{{< tabs name="self_subject_attributes_review_Example_1" >}}
+{{% tab name="table" %}}
+```
+ATTRIBUTE  VALUE
+Username   janedoe@example.com
+UID        b79dbf30-0c6a-11ed-861d-0242ac120002
+Groups     [developers system:authenticated]
+```
+{{% /tab %}}
+
+{{% tab name="JSON" %}}
+```json
+{
+  "username": "janedoe@example.com",
+  "uid": "b79dbf30-0c6a-11ed-861d-0242ac120002",
+  "groups": ["developers", "system:authenticated"]
+}
+```
+{{% /tab %}}
+
+{{% tab name="YAML" %}}
+```yaml
+username: janedoe@example.com
+uid: b79dbf30-0c6a-11ed-861d-0242ac120002
+groups:
+- developers
+- system:authenticated
```

Comment:
If there are extra data present, do these show up? If so, maybe we need a simple example and a more complicated example.

Reply:
I added a simple and a complicated example.